### ISSUE TITLE
where is the mask_rcnn_dbswin-t_fpn_3x_nuim_cocopre.pth?

### ISSUE THREAD

**Issue by fdy61** [hht1996ok/EA-LSS#24]

**Comment by Da1symeeting1** [comment]
@fdy61  Have you get the pretrained model? I need it too, if you can send it to me plz send it to 1241955089@qq.com plz. Thanks for that ,looking forward to your reply.

**Comment by fdy61** [comment]
No, I haven't get it. The author doesn't send me the pretrained model.

**Comment by hht1996ok** [comment]
@Da1symeeting1 @fdy61  Weights can be found at the following links: https://drive.google.com/file/d/1BcHPxVv_C1LC8oRc-vBhbMk2R6zDYfFx/view?usp=sharing

**Comment by fdy61** [comment]
@hht1996ok Thank you.

**Comment by Da1symeeting1** [comment]
@hht1996ok  Thank you !!

**Comment by WenxuanLi-whu** [comment]
你这个是自己训练的权重还是套用别人的啊，精度多少来着，这个玩意影响很大的

**Comment by Da1symeeting1** [comment]
> 你这个是自己训练的权重还是套用别人的啊，精度多少来着，这个玩意影响很大的

这个是bevfusion（peiking）预训练，用这个我复现bevfusion cam mAP是34.78左右
